src/Microsoft.SqlTools.ServiceLayer/ExecutionPlan/Contracts/ExecutionPlanGraph.cs

```diff
@@ -189,4 +198,12 @@ public enum BadgeType
         CRITICALWARNING = 1,
         PARALLELISM = 2
     }
+
+    public enum PropertyValueDataType
+    {
+        NUMBER = 0,
```

Comment:
typically in C# naming convention enum values use PascalCasing (first char cap and subsequent work breaks are cap, rest lower-case), all caps is more C-style.